tests/security/swtpm/tpm_selftest.pm

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Update TrouSerS and tpm-tools to the latest version
+#          Make sure your setup has a TPM 1.2 device attached,
+#          both hardware or software devices should be fine
+#          We need only test this feature on aarch64 platform,
+#          However, based on bsc#1193350, test it on x86_64 as
+#          well
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#103644, tc#1769832
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils qw(zypper_call systemctl);
+use Utils::Architectures;
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Version check
+    my $pkg_list = {'tpm-tools' => '1.3.9.2', trousers => '0.3.15'};
+    foreach my $pkg (keys %$pkg_list) {
+        zypper_call("in $pkg");
```

Comment:
why not install these two packages in one time? I remember you have a pr to do that.

Reply:
Good comment, let me fix it